LLAP/llap.sh

```diff
@@ -0,0 +1,478 @@
+    sed -i 's/print \"%s Packaged the files\" % (strftime(\"%H:%M:%S\", gmtime()))/print(\"%s Packaged the files\" % (strftime(\"%H:%M:%S\", gmtime())))/g' /usr/lib/hive/scripts/llap/yarn/package.py
+    sed -i 's/0700/0o700/g' /usr/lib/hive/scripts/llap/yarn/package.py
+    sed -i 's/print \"%s Prepared %s\/run.sh for running LLAP on YARN\" % (strftime(\"%H:%M:%S\", gmtime()), output)/print(\"%s Prepared %s\/run.sh for running LLAP on YARN\" % (strftime(\"%H:%M:%S\", gmtime()), output))/g' /usr/lib/hive/scripts/llap/yarn/package.py
+    sed -i 's/long(max_direct_memory)/int(max_direct_memory)/g' /usr/lib/hive/scripts/llap/yarn/package.py
+}
+
+##if the metadata value exists, then we want to configure the local ssd as a caching location.
+function configure_SSD_caching_worker(){
+    if [[ -n "$HAS_SSD" ]]; then
+        echo "ssd"
+        mkdir /mnt/1/llap
+        chmod 777 /mnt/1/llap
+        chown hive:hive /mnt/1/llap
+        
+        bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.llap.io.allocator.mmap' --value 'true' \
+        --clobber
+        bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.llap.io.allocator.mmap.path' --value '/mnt/1/llap' \
+        --clobber
+        bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.llap.io.memory.mode' --value 'cache' \
+        --clobber
+        bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.llap.io.use.lrfu' --value 'true' \
+        --clobber
+    else
+        echo "NO SSD to configure..."
+    fi
+}
+
+##if the metadata value exists, then we want to configure the local ssd as a caching location. We don't need to make any directory changes since master nodes don't 
+##run YARN. We do need to ensure that the configuration files are equal across master and worker nodes
+function configure_SSD_caching_master(){
+    if [[ -n "$HAS_SSD" ]]; then
+	   echo "ssd"
+
+        bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.llap.io.allocator.mmap' --value 'true' \
+        --clobber
+        bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.llap.io.allocator.mmap.path' --value '/mnt/1/llap' \
+        --clobber
+        bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.llap.io.memory.mode' --value 'cache' \
+        --clobber
+        bdconfig set_property \
+        --configuration_file "${HIVE_CONF_DIR}/hive-site.xml" \
+        --name 'hive.llap.io.use.lrfu' --value 'true' \
+        --clobber
+    else
+        echo "NO SSD to configure..."
+fi
+}
+
+##start LLAP - Master Node
+function start_llap(){
+
+
+	if [[ "${HOSTNAME}" == "${LLAP_MASTER_FQDN}" ]]; then
```

Comment:
indentation not consistent with rest of the file.

Reply:
done